packages/governance/src/tests/test_fuzz_packing.cairo

```diff
@@ -0,0 +1,38 @@
+use crate::governor::ProposalCore;
+use crate::multisig::storage_utils::TxInfo;
+use starknet::storage_access::StorePacking;
+
+#[test]
+fn test_pack_unpack_tx_info(is_executed_val: u8, submitted_block: u64) {
+    let is_executed = is_executed_val % 2 == 0;
+    let packed_value = StorePacking::pack(TxInfo { is_executed, submitted_block });
+    let unpacked_tx_info: TxInfo = StorePacking::unpack(packed_value);
+
+    assert_eq!(unpacked_tx_info.is_executed, is_executed);
+    assert_eq!(unpacked_tx_info.submitted_block, submitted_block);
+}
+
+#[test]
+fn test_pack_unpack_proposal_core(
+    proposer_val: felt252,
+    vote_start: u64,
+    vote_duration: u64,
+    executed_val: u8,
```

Comment:
We can implement Fuzzable for bool as described [here](https://foundry-rs.github.io/starknet-foundry/appendix/snforge-library/fuzzable.html) with 0.38.0. Let's use that for our fuzz tests types (applies for other PRs as well)